platform/drm/meson.build

```diff
@@ -0,0 +1,21 @@
+drm_platform_plugin = shared_module('cogplatform-drm',
+    'cog-platform-drm.c',
+    'cog-drm-renderer.c',
+    'cog-drm-gles-renderer.c',
+    'cog-drm-modeset-renderer.c',
+    'kms.c',
+    'cursor-drm.c',
+    c_args: ['-DG_LOG_DOMAIN="Cog-DRM"'],
+    dependencies: [
+        cogcore_dep,
+        wpebackend_fdo_dep,
+        dependency('epoxy'),
+        dependency('libdrm', version: '>=2.4.71'),
+        dependency('libinput'),
+        dependency('libudev'),
+    ],
+    gnu_symbol_visibility: 'inlineshidden',
```

Comment:
`'inlineshidden'` is a combination of `'-fvisibility=hidden'` (C/C++) and `'-fvisibility-inlines-hidden'` (C++ only). I'm not sure you need the latter?

Meson 0.63.0 learned to demote `'inlineshidden'` to plain old `'hidden'` for C source files. Older versions of Meson would emit the following warning when running `ninja` and running the C compiler:

```
cc1: warning: command-line option ‘-fvisibility-inlines-hidden’ is valid for C++/ObjC++ but not for C
```

The Meson change is because it's possible to have mixed mode C/C++ sources in a single target. :)